cmd/tckconvert.cpp

```diff
@@ -102,13 +108,17 @@ void usage ()
 
 class VTKWriter: public WriterInterface<float> { MEMALIGN(VTKWriter)
   public:
-    VTKWriter(const std::string& file) : VTKout (file) {
+    VTKWriter(const std::string& file, bool write_binary = false) : VTKout (file, std::ios::binary ), write_binary(write_binary){
       // create and write header of VTK output file:
       VTKout <<
-        "# vtk DataFile Version 1.0\n"
-        "Data values for Tracks\n"
-        "ASCII\n"
-        "DATASET POLYDATA\n"
+        "# vtk DataFile Version 4.2\n"
```

Comment:
I note the version number has changed. Is that necessary for the binary output, and does that impact on the ASCII output in any way...?

Reply:
I looked into this a bit more, the VTK documentation is conflicting.  The [Users Guide](https://www.kitware.com/products/books/VTKUsersGuide.pdf) on pg 470 says the current version is `3.0`.  I updated the code to write `3.0` (the 4.2 came from [this document, i.e. the first google hit](https://vtk.org/wp-content/uploads/2015/04/file-formats.pdf)).